module/motion/QuinticWalk/data/config/webots/QuinticWalk.yaml

```diff
@@ -3,20 +3,20 @@ log_level: WARN
 walk:
   # Full walk cycle frequency (how long it takes in seconds for a left + right steps)
   # (in Hz, > 0)
-  freq: 2.75
+  freq: 2.882156171492855
   # Length of double support phase in half cycle (how long both legs should be on the ground, as a ratio of how long it takes for a single step)
   # (ratio, [0:1])
-  double_support_ratio: 0.1
+  double_support_ratio: 0.02126588783229817
   # Give extra swing to first step for better start
   # (in ?, [0:10])
-  first_step_swing_factor: 0.8 #1.0
+  first_step_swing_factor: 1.0 #1.0
   foot:
     # Lateral distance between the feet center (how spread apart the feet should be)
     # (in m, >= 0)
-    distance: 0.18 #0.28
+    distance: 0.16573612838905935 #0.28
     # Maximum flying foot height (how high we lift a foot from the ground)
     # (in m, >= 0)
-    rise: 0.2 #0.1
+    rise: 0.06696921459985106 #0.1
     # Pause of Z movement on highest point (how long we hold a foot at the top, as a ratio of how long we're standing on one foot)
     # (single support cycle ratio, [0,1])
     z_pause: 0.0 #0.04
```

Comment:
Remove the rest of the comment ones _IMO_. If they've survived this long as comments, scrap 'em. People can look in the git history if they want